### DISCUSSION TITLE
Nova Impersonatable is not working

### DISCUSSION THREAD

**Discussion by ZeyadHawash9** [laravel/nova-issues#6177] [Q&A]
- Laravel Version: ^10.10
- Nova Version: 4.32.11
- PHP Version: ^8.1
- Database Driver & Version:
- Operating System and Version: #.#
- Browser type and version: #.#
- Reproduction Repository: https://github.com/###/###

### Description:
![112](https://github.com/laravel/nova-issues/assets/148364616/daece49f-08bc-4a68-9e50-a02f78cabf7f)

**Comment by bm2ilabs** [comment]
this issue still presist
